react/AddProductBtn.tsx

```diff
@@ -27,6 +27,24 @@ interface AddProductBtnState {
 
 const ICON_SIZE_SMALL = 16
 const ICON_SIZE_LARGE = 32
+const messages = defineMessages({
```

Comment:
Extract all this messages in a file in `constants/messages.js` or `utils/messages.js` and import it and use :smile:

Reply:
Nooo

This messages will only have one dependency. You don't need to extract something to another file if this thing will not be used in another module.

Reply:
It's called colocation. Keep things that belong to the component together.

This way you don't have to hop between files when changing the `AddProductBtn`

And when we eventually delete this file we don't need to go to other files and delete the messages.

Reply:
@klzns, Normally theres some messages being reused in multiple components (message.myLists is an example in this app).
Also, adding this messages in each Component does not look clean to me (but this is just a personal opinion).

Reply:
If it's shared then it's fine, but I would rather keep what's being used by a single instance in the same file.